larpandoracontent/LArHelpers/LArMvaHelper.h

```diff
@@ -1,26 +1,57 @@
 /**
- *  @file   larpandoracontent/include/LArHelpers/LArSvmHelper.h
+ *  @file   larpandoracontent/include/LArHelpers/LArMvaHelper.h
  *
- *  @brief  Header file for the lar svm helper class.
+ *  @brief  Header file for the lar mva helper class.
  *
  *  $Log: $
  */
-#ifndef LAR_SVM_HELPER_H
-#define LAR_SVM_HELPER_H 1
-
-#include "larpandoracontent/LArObjects/LArSupportVectorMachine.h"
+#ifndef LAR_MVA_HELPER_H
+#define LAR_MVA_HELPER_H 1
 
 #include <fstream>
 #include <chrono>
 #include <ctime>
 
+class AdaBoostDecisionTree;
+class SupportVectorMachine;
```

Comment:
Looks like these aren't needed (they aren't in namespace lar_content and everything still builds)

Reply:
Removed.